commands/zrange.md

```diff
@@ -51,16 +51,34 @@ ZRANGE zset (5 (10 BYSCORE
 
 Will return all the elements with `5 < score < 10` (5 and 10 excluded).
 
+## Rev ranges
+
+Using the `REV` option reverses the sorted set, with index 0 as the element with the highest score. In this case, `<start>` must be less than or equal to `<stop>` to return anything. However, if the `BYSCORE` option is selected, the `<start>` is the highest score to consider, and `<stop>` is the lowest score to consider, therefore `<start>` must be less than or equal to `<stop>`
```

Comment:
Line breaks between every sentense for new text. (See CONTRIBUTING.md in this repo.)

Also `.` is missing in the end.

```suggestion
Using the `REV` option reverses the sorted set, with index 0 as the element with the highest score.
In this case, `<start>` must be less than or equal to `<stop>` to return anything.
However, if the `BYSCORE` option is selected, the `<start>` is the highest score to consider, and `<stop>` is the lowest score to consider, therefore `<start>` must be less than or equal to `<stop>`.
```